pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -144,6 +144,18 @@ func NewCloudCIDRAllocator(client clientset.Interface, cloud cloudprovider.Inter
 		DeleteFunc: nodeutil.CreateDeleteNodeHandler(ca.ReleaseCIDR),
 	})
 
+	nwInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: nodeutil.CreateAddNetworkHandler(func(network *networkv1.Network) error {
```

Comment:
shouldn't this been an issue for creation of nwInformer ?

Reply:
This will return safely, but keep the reflector erroring, always adding CRDs will resolve this.

Reply:
I don't understand why we use this `nodeutil.CreateAddNetworkHandler(` is complex to think about, instead of adding the function directly to the handler, is much easier to read and it seems that does the same